packages/pages/tsdoc-metadata.json

```diff
@@ -0,0 +1,11 @@
+// This file is read by tools that parse documentation comments conforming to the TSDoc standard.
+// It should be published with your NPM package.  It should not be tracked by Git.
```

Comment:
This comment sounds like we need to add this to .gitignore

Reply:
Yes! I wasn't suppose to allow the workflow to add + commit that file

Reply:
Wait, we don't want the workflow to commit this either. The message says `It should not be tracked by Git` so this file should be added to .gitgnore.

Reply:
It is being ignored by the workflow currently (not part of the files the workflow commits). This file shouldn't be present in current commit

Reply:
Got it 👍